ogn/parser/pattern.py

```diff
@@ -37,12 +38,15 @@
     RAM:(?P<ram_free>[\d.]+)\/(?P<ram_total>[\d.]+)MB\s
     NTP:(?P<ntp_offset>[\d.]+)ms\/(?P<ntp_correction>[+-][\d.]+)ppm\s?
     (?:(?P<cpu_temperature>[+-][\d.]+)C\s*)?
+    (?:(?P<aircraft_counter_visible>\d+)\/(?P<aircraft_counter_total>\d+)Acfts\[1h\]\s*)?
```

Comment:
Here is the complete regex (we took it from ogn-client-ruby), so I also parsing the RF-Block:

```
(?:
        v(?P<version>\d+\.\d+\.\d+)
        \.?(?P<platform>.+?)?
    \s)?
    CPU:(?P<cpu_load>[\d.]+)\s
    RAM:(?P<ram_free>[\d.]+)\/(?P<ram_total>[\d.]+)MB\s
    NTP:(?P<ntp_offset>[\d.]+)ms\/(?P<ntp_correction>[+-][\d.]+)ppm\s?
    (?:(?P<cpu_temperature>[+-][\d.]+)C\s*)?
    (?:(?P<aircraft_counter_visible>\d+)\/(?P<aircraft_counter_total>\d+)Acfts\[1h\]\s*)?
    (?:RF:
        (?:
            (?P<manual_correction>[+-][\d]+)
            (?P<automatic_correction>[+-][\d.]+)ppm\/
        )?
        (?P<input_noise>[+-][\d.]+)dB
        (?:\/(?P<total_snr>[+-][\d.]+)dB\@10km\[(?P<total_fixes>\d+)\])?
        (?:\/(?P<daily_snr_selection>[+-][\d.]+)dB\@10km\[(?P<daily_devices_selection>\d+)\/(?P<daily_devices>\d+)\])?
    )?
```

I asked @pjalocha about the protocol and got this:
`HochschDA>APRS,TCPIP*,qAC,GLIDERN1:>175118h v0.2.5.ARM CPU:0.4 RAM:638.0/970.5MB NTP:0.2ms/-1.1ppm +65.5C 14/16Acfts[1h] RF:+45+0.0ppm/+3.88dB/+24.0dB@10km[143717]/+26.7dB@10km[68/135]`

> 14/16Acfts[1h]
> means 16 aircrafts received within the last hour, and 14 of these were visible, while 2 were either no-track or been made invisible by the DB.
> 
> +24.0dB@10km[143717]/+26.7dB@10km[68/135]
> is meant to measure the sensitivity, I measure the signal SNR vs distance and transmitter power
> and then I normalize to 10km distance.
> 
> The first part means the average SNR of all signals was +24dB when normalized to 10km distance and 10dBm
> in square brackets you have the number of signals incliuded.
> This number is calcaluted since the station has started thus it can include data from several days or weeks.
> 
> The seconds part is when you select the "better half" thus it attempts to remove gliders/aircrafts with bad FLARM installation.
> In brackets you have number of aircrafts taken into this average and then the total number of aircrafts received.
> This number is calcaulted over 24 hour.